trace-utils/src/trace_utils.rs

```diff
@@ -450,6 +661,88 @@ mod tests {
         assert!(coalesced.len() > 1 && coalesced.len() < 5);
     }
 
+    #[tokio::test]
+    #[allow(clippy::type_complexity)]
+    async fn v5_get_traces_from_request_body() {
+        let data: (
+            Vec<String>,
+            Vec<
+                Vec<(
+                    u8,
+                    u8,
+                    u8,
+                    u64,
+                    u64,
+                    u64,
+                    i64,
+                    i64,
+                    i32,
+                    HashMap<u8, u8>,
+                    HashMap<u8, f64>,
+                    u8,
+                )>,
```

Comment:
I'm not familiar with the domain here--is there a struct/proto or something I can look at to see what each slot of the tuple means? At a first glance, I am inclined to agree with `clippy::type_complexity`, although I realize this is just in a test.

Reply:
great question, I wish I had one!
What I did find was this [documentation](https://github.com/DataDog/datadog-agent/blob/579d60cbfc8635559338b19fa855bf59b6d10f00/pkg/trace/api/version.go#L110) and finally this [test](https://github.com/DataDog/datadog-agent/blob/main/pkg/proto/pbgo/trace/decoder_v05_test.go).

I used the same fields to create the test to confirm this all worked (as well as implementing it using the python tracer).

Reply:
That doc comment is unfortunately the reference documentation.